index.html

```diff
@@ -228,5 +228,37 @@ <h3>Languages</h3>
             </div>
         </div>
     </div>
-</body>
-</html>
\ No newline at end of file
+    <form class="former-block">
+                <div class="form-block">
+                    <section>
+                        <h1 class="contact">Contact me</h1>
+                            <p class="paragraph">If you have an application y ou are interested in developing,
+                                a feature that you need built or   a project that needs 
+                                coding. I’d love to help with it 
+                            </p>
+                        </section>
+                            <section>
+                                  <ul class="form-container">
+                                    <li class="input-name input-email-li">
+                                        <input  class="input-email" type="text" name="name" id="name" maxlength="30" placeholder="safari hamuli" required>
+                                    </li>
+                                    <li class="input-email-li">
+                                        <input class="input-email" type="email" name="email" id="email" placeholder="safari@gmail.com" required>
+                                        
+                                    </li>
+                                    <li class="input-email-li">
+                                        <textarea class="text-area" name="text_area" id="text_area" cols="30" rows="10" maxlength="500"
```

Comment:
- [ ] Kindly apply `font-family` to this input field to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200882592-467ce694-114c-4633-ac01-9986554982e0.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200882657-def20a2a-8148-4e67-b743-1500f0191a88.png)